internal/commands/parameters.go

```diff
@@ -85,14 +85,21 @@ func withStrictMode(strict bool) mergeBundleOpt {
 		return nil
 	}
 }
+
+func withImmutableParameters(verify bool) mergeBundleOpt {
```

Comment:
I wonder if we really want this to be configurable, as it could be forgotten in the futur and we clearly don't want immutable parameters to be overridden. 🤔

Reply:
After talking about that on Slack, it's needed to be able to check at upgrade but not during installation.